frontend/packages/shared/src/api/mutations.ts

```diff
@@ -167,3 +168,12 @@ export const updateProcessDataTypes = (org: string, app: string, dataTypesChange
 
 // Maskinporten
 export const updateSelectedMaskinportenScopes = (org: string, app: string, appScopesUpsertRequest: MaskinportenScopes) => put(selectedMaskinportenScopesPath(org, app), appScopesUpsertRequest);
+
+// Org level code lists - TODO: Replace mocks with correct paths.
+export const createOrgLevelCodeList = async (codeListItem: CodeList): Promise<void> =>
```

Comment:
Denne bør også ta imot `org` og navn på kodeliste som parametre.

Her kan vi for øvrig godt bruke `OptionList` som type, så det blir likt som på appnivå. På appnivå er det ikke noe `POST`-endepunkt der siden `PUT`-endepunktet brukes til både opprettelse og oppdatering, men de to endepunktene skal oppføre seg helt likt. Se [Slack-diskusjon](https://digdir.slack.com/archives/C077KTTCBSS/p1737715900839899).